transform/snowflake-dbt/models/mattermost/hourly/focalboard_user_retention.sql

```diff
@@ -45,4 +45,17 @@ GROUP BY first_active.first_active_timestamp
   ,   first_active.user_actual_id 
   ,   first_active.installation_id 
   ,   first_active.first_server_edition 
-  ,   first_active.id
\ No newline at end of file
+  ,   first_active.id
+) SELECT DISTINCT
```

Comment:
can we use a dbt formatter?
)
SELECT
  DISTINCT q2.first_active_timestamp,
  q2.server_id,
  q2.user_id,
  q2.installation_id,
  q2.first_server_edition,
  q2.id,
  q2.age,
  q2.max_original_timestamp,
  fet3.type AS TYPE
FROM
  q2 q2
  LEFT JOIN {{ ref('focalboard_event_telemetry') }}
  fet3
  ON fet3.user_actual_id = q2.user_id